Source/lua/randomat2/events/communist.lua

```diff
@@ -0,0 +1,98 @@
+AddCSLuaFile()
+
+local EVENT = {}
+
+EVENT.Title = "Communism! Time to learn how to share..."
+EVENT.id = "communist"
+
+
+util.AddNetworkString("AlertTriggerFinal")
+util.AddNetworkString("alerteventtrigger")
+
+local function RenameWeps(name)
```

Comment:
Consider extracting this to a common function either in the Randomat namespace or the Event namespace so it can be shared with the "privacy" event